meetings/2017-01-25.md

```diff
@@ -0,0 +1,285 @@
+# Node Foundation CTC Meeting 2017-01-25
+
+## Links
+
+* **Audio Recording**: <https://www.youtube.com/watch?v=8RIHCe3sFLU>
+* **GitHub Issue**: [CTC#63](https://github.com/nodejs/CTC/issues/63)
+* **Minutes Google Doc**:
+  <https://docs.google.com/document/d/15CWMkzOEeXEGqqt6S7j9cTk787azjMX2sozAdYEXm0g>
+* _Previous Minutes Google Doc_:
+  <https://docs.google.com/document/d/1Er0gdfsakYIY3Oi7ZlnKuyHrGl4_PNqGVf2BWAs6dUk>
+
+
+## Present
+
+* Anna Henningsen @addaleax (CTC)
+* Bradley Meck @bmeck (observer/GoDaddy/TC39)
+* Сковорода Никита Андреевич @ChALkeR (CTC)
+* Colin Ihrig @cjihrig (CTC)
+* Jeremiah Senkpiel @Fishrock123 (CTC)
+* Fedor Indutny @indutny (CTC)
+* James M Snell @jasnell (CTC)
+* Josh Gavant @joshgav (observer/Microsoft)
+* Michael Dawson @mhdawson (CTC)
+* Brian White @mscdex (CTC)
+* Ali Ijaz Sheikh @ofrobots (CTC)
+* Michaël Zasso @targos (CTC)
+* Jenn Turner @renrutnnej (observer/Node.js Foundation)
```

Comment:
I was there for the last 15 minutes. But I'm OK if that means I don't get a `* Rich Trott @Trott (CTC)`.